Descriptions/Curation.md

```diff
@@ -0,0 +1,62 @@
+All solubility assays in ChEMBL contain an entry with the mentioned organism “Rattus norvegicus” - it is certainly not an appropriate entry for solubility predictive model development. We can also compare entries from “Description” - “Aqueous solubility at pH 7.4” and “Solubility in SGF at pH 1.8,” and they cannot be mixed either as they fit the model for a specific solvent and specific pH level. “Aqueous solubility at pH 7.4” and “Water solubility at pH 7.4” can be mixed.
+- In most cases, data is presented as a point measurement, though confidence intervals could also be provided, we eliminate data with wide deviations.
+Sometimes data is presented with relations like >=, <, =. Depending on the task, this data can be used or eliminated. For some biological activities and ADMET properties, data is provided for a certain molecular target (like CYPs, hERG etc.). It is best to know if the data is related to a common biological target, its subunit, or a mutant resistant to some action. Try to take it into account.
+- For some biological activities and ADMET properties data is provided for a certain molecular target (like CYPs, hERG etc.). It is best to know if data is related to a common biological target, its subunit or a mutant resistant to some action. Try to take it into account.
+
+## Structural Data Curation
+
+One of the most crucial steps is structural data curation. Though we can proceed without it, the absence of structural standardization may lead to undesirable model properties.
+
+Structural curation steps are:
+
+1. Removal of incorrect structures. We use RDKit to define if the structure is valid.
+2. Removal of structures with rare elements. If there is no specific obstacle to conserving structures with rare elements, we remove all structures containing:
+	>"R  ", "D  "  "He ", "Li ","Be ", "B  ",,
+	"Ne ", "Al ", "Si ", "Ar ", "Sc ", "Ti ", "V  ", "Cr ", "Co ",
+	"Ni ", "Ga ", "Ge ", "As ", "Se ",
+	"Kr ", "Rb ", "Sr ", "Y  ", "Zr ", "Nb ",
+	"Mo ", "Tc ", "Ru ", "Rh ", "Pd ", "Ag ", "Cd ",
+	"In ", "Sn ", "Sb ", "Te ", "Xe ", "Cs ",
+	"Ba ", "La ", "Ce ", "Pr ", "Nd ", "Pm ", "Sm ",
+	"Eu ", "Gd ", "Tb ", "Dy ", "Ho ", "Er ", "Tm ",
+	"Yb ", "Lu ", "Hf ", "Ta ", "W  ", "Re ", "Os ",
+	"Ir ", "Pt ", "Au ", "Hg ", "Tl ", "Pb ", "Bi ",
+	"Po ", "At ", "Rn ", "Fr ", "Ra ", "Ac ", "Th ",
+	"Pa ", "U  ", "Np ", "Pu ", "Am ", "Cm ", "Bk ",
+	"Cf ", "Es ", "Fm ", "Md ", "No ", "Lr ", "Rf ",
+	"Db ", "Sg ", "Bh ", "Hs ", "Mt ", "Ds ", "Rg ",
+	"Cn ", "Nh ", "Fl ", "Mc ", "Lv ", "Ts ", "Og "
+3. Taking the major component: The desired property usually depends on the single structure. Entries in the dataset representing mixtures and salts contain more than one structural component. This can significantly affect generated descriptors/fingerprints - mixtures and single molecules will have different sets of features.
```

Comment:
Please, decide which punctuation marks you use for separation)))
It is a mystery for me right now) I've already seen dots, hyphens and here is a colon (before you used hyphen)